go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/bitrise-steplib/steps-authenticate-with-bitbucket-oauth
+
+go 1.18
```

Comment:
We still have some stacks running with Go 1.17, so I would rather use 1.17 here, just in case.

Reply:
Good catch, thanks, updated in https://github.com/bitrise-steplib/steps-authenticate-with-bitbucket-oauth/pull/7/commits/f4a11b66a27d211bfefd7a0f8e30f7b55a87c6f1